dds/DCPS/ConfigStoreImpl.cpp

```diff
@@ -103,6 +104,11 @@ DDS::Boolean
 ConfigStoreImpl::get_boolean(const char* key,
                              DDS::Boolean value)
 {
+  if (DCPS_debug_level) {
```

Comment:
This seems like it should be higher.

Reply:
Ideally, the user would only see configuration messages at start-up and during runtime if they change something.  I thought it was better to set these low so the configuration would be more obvious when debugging.

Reply:
I ran DevGuide Messenger on this branch and `ConfigStore::get` lines make up almost half the lines in the publisher log (299/620). There are a lot at the beginning, but also at least once on each write. I really think getting a value should be much much higher. Going by the [`DCPS_debug_level` list](https://github.com/OpenDDS/OpenDDS/blob/72462b97d5ee53f18e699260406487ae419e2ffd/dds/DCPS/debug.h#L69-L74) something happening every writer should be least an 8, if not it's own category like the special transport or security ones. Getting a value also doesn't say the value, so it doesn't seem that useful other than saying something wanted that value.

Setting a value should probably be lower than that since it happens a lot less often (27/620). However a lot of it is duplicating other logging, like when transports dump their settings, so I'm not sure it should be `DCPS_debug_level>=1`. Maybe at least 2?

Reply:
I'll reproduce your experiment.  My goal is to _not_ have a get for every write.

I'll clean up the logging so that it prints the key and value for `DCPS_debug_level >= 2`.